### DISCUSSION TITLE
Handlebar like Quicklinks

### DISCUSSION THREAD

**Discussion by keptenkrems** [microsoft-search/pnp-modern-search#3876] [Ideas]
Hi Community, 
is it possible to add new Handlebar Template directly into the choice of PnP Search Result Webpart.

I am not a good programmer in doing html / css stuff, so for me and I think for a lot of other users it would be great to have more choices.
One example could be a handlebar like the webpart quicklinks. If it would be possible to create such a handlebar, it would be great.

If it is not an option to add more handlebar templates in the pnp Search Result Webpart - is it possible for someone to help me out to create a custom handlebar with options like we have it in the standard webpart quicklinks. 
So, cardfields could be
- a title (formatted as Url) --> the complete Item could also be clickable, instead of just the title.
- a icon from uifabricicons
- a User field for displaying a contact person
- a description with a limited amount of signs

Maybe it could created like the people handlebar with different sizes --> different Options to add.

- Size one: Just icon and Title (as link)
- Size two: size one + description
- Size three: size two + userfield
- Size four: size three + 1 custom field

The source to that could be repost pages. Reason for that is, that you can get the clicks on that quicklinks. Then you know how often which link was klicked.

What do you think of that option? Can someone help me with that?

Thanks in advance,
Tobias

**Comment by kasperbolarsen** [comment]
Tobias, can you upload a mockup for us to evaluate, please. The text description isn't sufficient

something like this?
![image](https://github.com/user-attachments/assets/dc5c4805-6fdf-4359-a089-59c8a23ad38a)

**Reply by keptenkrems** [reply]
Hi Kasper, 
thank you for your fast response. Yes, you nailed it almost... the link, i meant is the link on the title or on the complete item, as it is like the in the quicklink webpart.

here are some mockups from me with the details of four sizes:

![image](https://github.com/user-attachments/assets/8ee684e2-d9ee-47ce-92d5-c906ecb91f89)

I hope this is ok, that I used your user

**Reply by kasperbolarsen** [reply]
I am working on a blog post about this,  and a draft version of the layouts are available at https://github.com/microsoft-search/pnp-modern-search-layouts/tree/main/Results%2FHandlebars%2FQuick%20Links%20lookalikes

**Comment by kasperbolarsen** [comment]
I have uploaded 3 layouts to https://github.com/microsoft-search/pnp-modern-search-layouts/tree/main/Results/Handlebars/Quick%20Links%20lookalikes  ( I haven't finished the documentation yet )

they look like this
![image](https://github.com/user-attachments/assets/421bda21-501d-44ef-9878-f4c070810151)

**Reply by kasperbolarsen** [reply]
Documentation completed, https://github.com/microsoft-search/pnp-modern-search-layouts/blob/main/Results/Handlebars/Quick%20Links%20lookalikes/QuickLinks.md
